### DISCUSSION TITLE
[swc-loader] is react.refresh affected by webpack mode?

### DISCUSSION THREAD

**Discussion by theogravity** [swc-project/swc#4555] [Q&A] [answered]
I am using `swc-loader` for webpack, and have `react.refresh` option enabled in my `.swcrc` file. When I build for prod, I probably would want to disable it. I saw that `react.development` is affected by the webpack `mode` setting, and was wondering if this was as well or would I have to make another config specific to my prod build?

**Comment by Austaras** [comment, marked as answer]
Yes, react refresh is only enabled when `react.development` is true.
